pkg/log/checkpoint/checkpoint.go

```diff
@@ -23,20 +38,83 @@ func Run(logger log.Logger, db *bbolt.DB) {
 	//  * runtime stats, like memory allocations
 
 	go func() {
-		logCheckPoint(logger)
+		logCheckPoint(logger, db, opts)
 
 		for range time.Tick(time.Minute * 60) {
-			logCheckPoint(logger)
+			logCheckPoint(logger, db, opts)
 		}
 	}()
 }
 
-func logCheckPoint(logger log.Logger) {
-	logger.Log(
-		"msg", "log checkpoint started",
-		"hostname", hostName(),
-		"notableFiles", fileNamesInDirs(notableFileDirs...),
-	)
+func logCheckPoint(logger log.Logger, db *bbolt.DB, opts launcher.Options) {
+
+	logger.Log("msg", "log checkpoint started")
+
+	boltStats, err := agent.GetStats(db)
+	if err != nil {
+		logger.Log("bbolt db size", err.Error())
+	} else {
+		logger.Log("bbolt db size", boltStats.DB.Size)
+	}
+
+	logger.Log("hostname", hostName())
+	logger.Log("notableFiles", fileNamesInDirs(notableFileDirs...))
+
+	dialer := &net.Dialer{Timeout: requestTimeout}
+	ipLookuper := &net.Resolver{}
+	httpClient := &http.Client{Timeout: requestTimeout}
+
+	kolideServerUrl, err := parseUrl(opts.KolideServerURL)
+	if err != nil {
+		logCheckPointError(logger, errors.Wrap(err, "parsing URL"))
+	}
+
+	logMap(logger, testConnections(dialer, kolideServerUrl.Host))
+	logMap(logger, lookupHostsIpv4s(ipLookuper, kolideServerUrl.Host))
+
+	if opts.KolideHosted {
+		logMap(logger, fetchFromUrls(httpClient, kolideServerUrl.String()+"/version"))
+		logMap(logger, testConnections(dialer, "dl.kolide.co"))
+		logMap(logger, lookupHostsIpv4s(ipLookuper, "dl.kolide.co"))
```

Comment:
These should be in the `if opt.Autoupdate` block, and should use `opts.MirrorURL`

Reply:
just to make sure I understand ....

the comment for opts.MirrorURL states `MirrorServerURL is the URL for the Notary mirror.`

therefore I can assume that "dl.kolide.co" is the notary mirror when running kolide_hosted?

Reply:
`dl.kolide.co` is the default MirrorServerURL. It's often used even if not kolide_hosted. Historically, there's a bunch of people that use our TUF and Mirror servers for their onprem things.